src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -283,6 +282,7 @@ private void appendSearchResult(List<MapBean> maps, Pane pane) {
   }
 
   private void displayMapsFromSupplier(Supplier<CompletableFuture<List<MapBean>>> mapsSupplier) {
+    currentPage = 0;
```

Comment:
setting this here means that you can also click the more button if it was getNewest Replays and not only with a query